tutor/commands/jobs.py

```diff
@@ -315,6 +317,49 @@ def sqlshell(args: list[str]) -> t.Iterable[tuple[str, str]]:
     yield ("lms", command)
 
 
+@click.command(context_settings={"ignore_unknown_options": True})
+@click.option(
```

Comment:
This should not be an option, but an argument, because it's mandatory. The command signature should not be:

     tutor local do update_mysql_authentication_plugin --users=regis,danyal

but:

    tutor local do update_mysql_authentication_plugin regis danyal

Moreover, we should be able to update all users with users=all:

    tutor local do update_mysql_authentication_plugin all